plugins/aggregators/basicstats/basicstats.go

```diff
@@ -2,26 +2,31 @@ package basicstats
 
 import (
```

Comment:
We will have to create our own aggregator plugin, given the extensive changes. We cannot change the behavior for other users so drastically. Please add another plugin and copy whatever code is needed from this file and revert this file to the original.

Reply:
Agree. Writing a separate aggregator plugin.

Reply:
reverted the changes done in basicstats.go to the commit from where i forked my branch.

Reply:
added a separate file with the required functionalities.